app/assets/stylesheets/application.css.scss

```diff
@@ -11,7 +11,6 @@
  * file per style scope.
  *
  *= require_self
- *= require framework_and_overrides
```

Comment:
should we / can we, remove all the old compass imports from this file as well?

Reply:
yes indeed